stdlib/src/builtin/hash.mojo

```diff
@@ -174,13 +174,12 @@ fn _hash_simd[type: DType, size: Int](data: SIMD[type, size]) -> Int:
     var final_data = bitcast[int_type, 1](hash_data[0]).cast[DType.uint64]()
 
     @parameter
-    fn hash_value[i: Int]():
+    for i in range(1, size):
```

Comment:
shouldn't this be 
```suggestion
    for i in range(size-1):
```

Reply:
I changed the use site from `i + 1` to `i`.